assets/js/data/payment/selectors.ts

```diff
@@ -105,6 +106,13 @@ export const expressPaymentMethodsInitialized = (
  * isPaymentFinished, hasPaymentError, isPaymentSuccess, isPaymentFailed
  */
 export const getCurrentStatus = ( state: PaymentMethodDataState ) => {
+	deprecated( 'getCurrentStatus', {
```

Comment:
Please could you add a link to this PR in the deprecated message?

Reply:
Like here: https://github.com/woocommerce/woocommerce-blocks/blob/4fcb6b28d69903b00dd296dec08db32a8d5ee67a/packages/checkout/filter-registry/index.ts#L66

Reply:
Done :)